style.css

```diff
@@ -80,3 +80,262 @@ nav {
   align-items: center;
   list-style-type: none;
 }
+
+.works {
+  margin-top: 114px;
+  margin-left: 24px;
+  margin-right: 24px;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  grid-row-gap: 88px;
+}
+
+.works-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 12px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.card-img {
+  margin-top: 16px;
+  margin-left: 16px;
+  margin-right: 16px;
+  width: 295px;
+  height: 220px;
+}
+
+.card1 {
+  background: url(img/Snapshoot-Portfolio1.png);
+}
+
+.card2 {
+  background: url(img/Snapshoot-Portfolio2.png);
+}
+
+.card3 {
+  background: url(img/Snapshoot-Portfolio3.png);
+}
+
+.card4 {
+  background: url(img/Snapshoot-Portfolio4.png);
+}
+
+.card-title {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-top: 12px;
+  margin-left: 16px;
+}
+
+.card-details {
+  display: flex;
+  align-items: center;
+  margin-top: 20px;
+  list-style: none;
+}
+
+.company {
+  margin-right: 12px;
+  margin-left: 16px;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.circle {
+  margin-right: 12px;
+}
+
+.job {
+  margin-right: 12px;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.year {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.card-info {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 20px;
+  margin-right: 16px;
+  margin-left: 16px;
+}
+
+.tags {
+  display: flex;
+  margin-top: 12px;
+  list-style: none;
+  margin-left: 16px;
+}
+
+.tag {
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  margin-right: 8px;
+  color: #6070ff;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.btn {
+  color: #396df2;
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background: none;
+  margin-bottom: 28px;
+}
+
+.card-button {
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2;` and on hover has the `background: #6070FF;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```